### DISCUSSION TITLE
>2.8.2 prerelease?

### DISCUSSION THREAD

**Discussion by blast-hardcheese** [pydantic/pydantic#10183] [Question] [answered]
Heya, I'm pulling in a recent source release so I can get https://github.com/pydantic/pydantic-core/pull/1332, since `pydantic-core` is tightly coupled.

I'm happy to keep using source releases, but it prevents publishing new package versions to pypi.

What kind of timeframe should we expect for the next prerelease or full release?

Thank you in advance!

**Comment by Viicos** [comment, marked as answer]
We expect a 2.9 beta release early next week

**Comment by blast-hardcheese** [comment]
Thank you kindly!

**Reply by Viicos** [reply]
Checklist: https://github.com/pydantic/pydantic/issues/10189
